File: C#/FindOdd.cs
//6 kyu
//https://www.codewars.com/kata/54da5a58ea159efa38000836  
  
  class Kata
    {
    public static int find_it(int[] seq) 
      {
       int count = 0;
  for(int i = 0; i<seq.Length; i++){
    for(int j = 0; j<seq.Length; j++){
      if(seq[i] == seq[j]){
        count++;
      }
    }
    if(count % 2 != 0 ){
      return seq[i];
    }
  }
      return -1;
      }
       
    }

File: Java/FindOdd.java
//6 kyu
//https://www.codewars.com/kata/54da5a58ea159efa38000836  
import java.util.HashMap;
import java.util.Map;

public class FindOdd {
	public static int findIt(int[] a) {
  	HashMap<Integer, Integer> map = new HashMap<>();
        for (int number : a) {
            int count = map.getOrDefault(number, 0);
            map.put(number, count + 1);
        }
        for (Map.Entry<Integer, Integer> entry : map.entrySet()) {
            if (entry.getValue() % 2 != 0) return entry.getKey();
        }
        return 0;
  }
}